### PR TITLE
20092 e2e payment request fixture

### PR DESCRIPTION
**Pull request by nnorbert**
Payment request fixture for E2E testing.

### CODE REVIEW

File: apps/payments/e2e/fixtures/paymentRequestsFixtures.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { PaymentRequestFormPage } from "../objects/paymentRequests/PaymentRequestFormPage";
+import { test as base } from "./providersFixtures";
+
+type paymentRequestsFixtures = {
+  paymentRequestMultiple: string;
+  paymentRequestRealex: string;
+};
+
+export const test = base.extend<paymentRequestsFixtures>({
+  paymentRequestMultiple: async (
```
Comment (alfonsograziano) [inline apps/payments/e2e/fixtures/paymentRequestsFixtures.ts:10]: Just as a suggestion, shall we rename this to `paymentRequestWithMultipleProviders`?

File: apps/payments/e2e/fixtures/paymentRequestsFixtures.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    { page, bankTransferProvider, openBankingProvider, stripeProvider },
+    use,
+  ) => {
+    const paymentRequestTitle = `Test multiple providers ${Date.now()}`;
+    const paymentRequestCreatePage = new PaymentRequestFormPage(page);
+    await paymentRequestCreatePage.goto();
+    await paymentRequestCreatePage.create({
```
Comment (alfonsograziano) [inline apps/payments/e2e/fixtures/paymentRequestsFixtures.ts:17]: Isn't the amount required? I guess otherwise we have a fallback to a default value?

**Review by alfonsograziano — DISMISSED** [review]
LGTM

**Review by peschina — APPROVED** [review]
LGTM

**Pull request merged**